packages/terra-disclosure-manager/src/terra-dev-site/doc/example/DisclosureComponent.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
+import { injectIntl, intlShape } from 'terra-base';
```

Comment:
We need to import `injectIntl, intlShape` this from `react-intl`. Pulling from terra-base ends up executing more of terra-base than just the `injectIntl` and `intlShape`. 
See https://github.com/cerner/terra-core/issues/1982 for more info.